### PR TITLE
feat(genai,#12654): FT-06 LoRA vision-langage Qwen3.5-0.8B — creation

### PR DESCRIPTION
**Pull request by jsboige**
Grain: DEEP/notebook-python — lane myia-po-2024:CoursIA — prev: MED/notebook-python #12663

See #12654

## Summary

Tranche 2 de #12654 : création de FT-06 — LoRA vision-langage sur Qwen3.5-0.8B, premier notebook vision de la série FineTuning. Tâche image→texte bornée et auto-contenue : formes géométriques colorées synthétiques (PIL, 3 formes x 4 couleurs, 24 images train + 6 held-out), contrat de sortie structuré `[IMG] forme=<...> couleur=<...> [FIN]` — le fil rouge "contrat de format" de la série (FT-03), transposé du texte pur au conditionnement par image.

Comparatif mesuré base vs LoRA-adapté (conformité de format + exactitude par champ forme/couleur), LoRA ciblant le décodeur (attention hybride + MLP, même cible que PT-02/FT-03), tour de vision laissé gelé (le contrat est un comportement de langage — cellule dédiée l'explique, l'extension vision = exercice 3).

## Validation

- Papermill end-to-end kernel `ft03-gpu` (torch 2.13.0+cu126, transformers 5.15.1, peft 0.20.0, torchvision 0.28.0 installé pour `AutoProcessor`) : 37/37 cellules, RC=0.

```
Executing: 100%|██████████| 37/37 [06:14<00:00, 10.11s/cell]
transplant OK: 14/14 code cells executed, 0 errors
scanned=6 notebooks_with_hits=0 occurrences=0
```

- 0 erreur volontaire (C.1), 3 exercices stubbés (`pass`/`print`), densité pédagogique **1692** chars md/cellule code (seuil 1200).
- Ratchet #11725 : filtres warnings cellule 1 (leçon FT-03) + `redirect_stdout` sur `AutoProcessor.from_pretrained` (le docstring-checker qwen3_vl de transformers 5.15 imprime des `[ERROR]` via `print` inconditionnel embarquant le chemin du venv — pas filtrable via `warnings`) → `scan_machine_path_outputs.py --check` = **0 occurrence**.

**Chiffres réels (greedy `do_sample=False`, éval déterministe)** :
- Base (avant LoRA) : conformité **0.00**, accuracy forme **0.00**, couleur **0.00** — mais les extraits bruts montrent que le base *décrit correctement les images en prose* (« un cercle bleu pur », « un triangle vert simple ») : défaillance purement contractuelle, pas perceptuelle.
- LoRA (après, 6 held-out jamais vues) : conformité **1.00**, forme **1.00**, couleur **1.00** — contrat ET lecture structurée installés.
- Entraînement : 7 274 496 params entraînables (0,8456 %), 308,4 s (5,1 min), loss 1,7677, VRAM max 1452 Mo sur RTX 3070 8 Go.
- Thèse decoder-only validée : la tour de vision gelée suffisait (le base voyait déjà) — le LoRA a appris au décodeur à lire les traits dans le format du contrat.

## Scope

2 fichiers :
- `MyIA.AI.Notebooks/GenAI/FineTuning/FT-06-Vision-Language-LoRA.ipynb` (création)
- `MyIA.AI.Notebooks/GenAI/FineTuning/README.md` (ligne FT-06 arbre + tableau)

## Suivi (hors scope, documentés)

- #12654 reste ouverte (tranche 1 = #12663 migration FT-03, cette PR = tranche 2 création FT-06 ; acceptation complète = les deux mergées).

### CODE REVIEW

**Comment by jsboige** [comment]
## Gate coordinateur §D — vérifié mécaniquement, pas repris du body (H.4)

Notebook téléchargé depuis ton head et mesuré, avec le dénominateur imprimé
pour qu'un vert par vacuité soit impossible :

```
FT-06-Vision-Language-LoRA.ipynb
  cellules code : 14   markdown : 23      <-- denominateur
  execution_count NULL      : 0
  outputs de type error     : 0
  erreurs volontaires (C.1) : 0
  execution_count contigus 1..N : True (max=14)
```

Une cellule code non vide sans output (#4) — je l'ai ouverte plutôt que de la
compter comme un output strippé : ce sont des **définitions de fonctions**
(`generer_vlm`, métriques), dont la dernière ligne significative est
`return metriques, lignes`. Une cellule qui ne termine pas sur une expression
n'imprime rien : silence légitime, pas sortie retirée.

**Label `machine-dep-timing-delta: +2`** : advisory, jamais bloquant
(`machine-dep-timing-advisory.yml` L112). Ton body traite d'ailleurs la cause
sœur — le `redirect_stdout` sur `AutoProcessor.from_pretrained`, parce que le
docstring-checker qwen3_vl de transformers 5.15 imprime en `print`
inconditionnel et n'est donc pas filtrable par `warnings`. C'est un diagnostic
de cause, pas un scrub de sortie : exactement ce que Stop & Repair demande.

**G-VAR-3 — même genre que ton grain précédent** (`notebook-python` après
`notebook-python` #12663), donc je dois justifier de ne pas HOLD plutôt que
laisser passer en silence. #12663 est une **migration de modèle** sur un
notebook SFT existant (OPT-1.3b → Qwen3.5-0.8B, +530/−313) ; #12668 **crée**
un notebook multimodal (+1275/−0, run papermill 37/37 cellules, RC=0). Le
litmus décisif — « pourrais-je en générer une douzaine en scannant l'instance
suivante ? » — est franchement négatif. Substance genuinement distincte : la
clause DEEP/MED de G-VAR-3 s'applique, pas le ban.

`DEEP/notebook-python` est un genre de **CONTENU** : ce grain tient le plancher
G-VAR-1 de ton cycle.

Merge.
